apps/data_handler/handlers/events/zklend/transform_events.py

```diff
@@ -50,8 +99,8 @@ class ZklendTransformer:
     A class that is used to transform Zklend events into database models.
     """
 
-    EVENT_MAPPING: Dict[str, Tuple[Callable, Type[BaseModel], Type[Base]]] = EVENT_MAPPING
-    PROTOCOL_ADDRESSES: str = ProtocolAddresses.ZKLEND_MARKET_ADDRESSES
+    EVENT_MAPPING: Dict[str, Tuple[Callable, str, Type[Base]]] = EVENT_MAPPING
+    PROTOCOL_ADDRESSES: str = "0x04c0a5193d58f74fbace4b74dcf65481e734ed1714121bdc571da345540efa05"
```

Comment:
Set constant variable just as a string it's really bad idea, because if I change this address I will need to update in every place where it's using.
that is why there was 
`ProtocolAddresses.ZKLEND_MARKET_ADDRESSES` then I can update only in one place and that's it